src/players/VAST.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react'
+import vast from 'vast-client'
+import { callPlayer } from '../utils'
+import createSinglePlayer from '../singlePlayer'
+import { FilePlayer } from './FilePlayer'
+
+const MATCH_URL = /^VAST:https:\/\//i
+export class VAST extends Component {
+  static displayName = 'VAST';
+  static canPlay = url => MATCH_URL.test(url);
+
+  state = {
+    sources: [],
+    tracker: null
+  }
+
+  callPlayer = callPlayer;
+
+  createSourceFiles (mediaFiles = []) {
+    return mediaFiles.map(({fileURL: src, mimeType: type}) => ({src, type}))
```

Comment:
is it possible this could contain empty entries? might de-structure a null right?

Reply:
addressed